### PR TITLE
extmod/network_ninaw10: Fix error messages.

### PR DESCRIPTION
**Pull request by iabdalkader**

### CODE REVIEW

File: extmod/network_ninaw10.c [hunk 1]
```diff
@@ -217,7 +217,7 @@ static mp_obj_t network_ninaw10_active(size_t n_args, const mp_obj_t *args) {
 
             if (fwver < fwmin) {
                 mp_raise_msg_varg(&mp_type_OSError,
-                    MP_ERROR_TEXT("Firmware version mismatch. Minimum supported firmware is v%d.%d.%d found v%d.%d.%d\n"),
+                    MP_ERROR_TEXT("firmware version mismatch. Minimum supported firmware is v%d.%d.%d found v%d.%d.%d\n"),
```
Comment (dpgeorge) [inline extmod/network_ninaw10.c:220]: I suggest "firmware version mismatch, minimum supported firmware is v%d.%d.%d found v%d.%d.%d" (ie no full stop, no trailing newline).

File: extmod/network_ninaw10.c [hunk 2]
```diff
@@ -197,14 +197,14 @@ static mp_obj_t network_ninaw10_active(size_t n_args, const mp_obj_t *args) {
             int error = 0;
             if ((error = nina_init()) != 0) {
                 mp_raise_msg_varg(&mp_type_OSError,
-                    MP_ERROR_TEXT("Failed to initialize Nina-W10 module, error: %d\n"), error);
+                    MP_ERROR_TEXT("failed to initialize Nina-W10 module, error: %d\n"), error);
```
Comment (dpgeorge) [inline extmod/network_ninaw10.c:200]: please remove the newline at the end (when errors are printed, newlines are added)

File: extmod/network_ninaw10.c [hunk 3]
```diff
@@ -197,14 +197,14 @@ static mp_obj_t network_ninaw10_active(size_t n_args, const mp_obj_t *args) {
...
             }
             // check firmware version
             uint8_t semver[NINA_FW_VER_LEN];
             if (nina_fw_version(semver) != 0) {
                 nina_deinit();
                 mp_raise_msg_varg(&mp_type_OSError,
-                    MP_ERROR_TEXT("Failed to read firmware version, error: %d\n"), error);
+                    MP_ERROR_TEXT("failed to read firmware version, error: %d\n"), error);
```
Comment (dpgeorge) [inline extmod/network_ninaw10.c:207]: please remove the newline

File: extmod/network_ninaw10.c (hunk 1 above)
Comment (iabdalkader) [reply extmod/network_ninaw10.c]: Okay.

**Comment by dpgeorge** [comment]
Thank you!

**Pull request merged**